### PR TITLE
deploying to vumigo

### PR DESCRIPTION
**Pull request by TrevorOctober**

### CODE REVIEW

**Comment by TrevorOctober** [comment]
@smn Please review cc @rudigiesler @justinvdm

File: docs/tutorial/deploying_to_VumiGo.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+Deploying to VumiGo
+===================
+
+In this part of the tutorial, we will learn how to deploy our app to `VumiGo <https://go.vumi.org>`_ using our sandbox skeleton app example.
```
Comment (smn) [inline docs/tutorial/deploying_to_VumiGo.rst]: it's two words `Vumi Go`

File: docs/tutorial/deploying_to_VumiGo.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+- Click new conversation
+
+.. figure::  images/conversation.png
+
+- Enter a new **conversation name** and **conversation description** and then select a **kind of conversation** of your choice. Please see the picture below. Click **save**.
+
+.. figure::  images/create_new_conversation.png
+
+- Now copy the code in `go-app.js <https://github.com/praekelt/go-jsbox-skeleton/blob/develop/go-app.js>`_ file and paste it here:
```
Comment (smn) [inline docs/tutorial/deploying_to_VumiGo.rst]: It's sometimes easier to just paste in the URL for where the go-app.js lives and have it loaded like that. Just remember that after you load the URL it'll show up in the editor but you still need to hit "save".

**Comment by smn** [comment]
some minor changes but other than that :+1:, thank you!

**Comment by TrevorOctober** [comment]
@smn Ready for re-review

File: docs/tutorial/deploying_to_VumiGo.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+In this part of the tutorial, we will learn how to deploy our app to `Vumi Go <https://go.vumi.org>`_ using our sandbox skeleton app example.
+
+**1. Set up a Vumi Go account**
+
+You will need a Vumi Go account to deploy our sandbox skeleton app. If you already have a Vumi Go account please move on to Step 2.
+
+- To set up a Vumi Go account please contact the vumi development team via email by joining the the `vumi-dev@googlegroups.com <https://groups.google.com/forum/?fromgroups#!forum/vumi-dev>`_ mailing list or on irc in #vumi on the `Freenode IRC network <https://webchat.freenode.net/?channels=#vumi>`_.
+
+**2. Sign in Vumi Go**
```
Comment (miltontony) [inline docs/tutorial/deploying_to_VumiGo.rst]: Sign in *to* Vumi Go

**Comment by miltontony** [comment]
👍 🍰

**Pull request merged**